ipaserver/install/service.py

```diff
@@ -231,6 +235,47 @@ def set_service_entry_config(name, fqdn, config_values,
         raise e
 
 
+def enable_services(fqdn):
+    """Change all configured services to enabled server
+
+    Server.ldap_enable() only marks a service as configured. Services
+    are enabled at the very end of installation.
```

Comment:
Can you expand on potential reasons why. I think the main purpose now is so that the services aren't advertised as available until the entire master is up and running.